ingest/src/main/java/com/connexta/ingest/adaptors/S3StorageAdaptor.java

```diff
@@ -38,20 +41,32 @@ public S3StorageAdaptor(final S3Client s3Client, final String s3BucketQuarantine
   }
 
   @Override
-  public void store(StoreRequest storeRequest, String key) throws IOException {
+  public void store(StoreRequest storeRequest, String key) throws IOException, StorageException {
     final String fileName = storeRequest.getFileName();
-
-    LOGGER.info("Storing {} in bucket \"{}\" with key \"{}\"", fileName, s3BucketQuarantine, key);
-    s3Client.putObject(
+    final PutObjectRequest putObjectRequest =
         PutObjectRequest.builder()
             .bucket(s3BucketQuarantine)
             .key(key)
             .contentType(storeRequest.getMimeType())
             .contentLength(storeRequest.getFileSize())
             .metadata(ImmutableMap.of("filename", fileName))
-            .build(),
+            .build();
+    final RequestBody requestBody =
         RequestBody.fromInputStream(
-            storeRequest.getFile().getInputStream(), storeRequest.getFile().getSize()));
+            storeRequest.getFile().getInputStream(), storeRequest.getFile().getSize());
+
+    try {
+      LOGGER.info("Storing {} in bucket \"{}\" with key \"{}\"", fileName, s3BucketQuarantine, key);
```

Comment:
✏️ Move this line out of the try catch. We don't want to catch RuntimeExceptions from the LOGGER.